### DISCUSSION TITLE
Upgrade vom piAlert -> NetAlertX

### DISCUSSION THREAD

**Discussion by antonov124** [netalertx/NetAlertX#631] [Q&A]
Hello, Is it possible to upgrade from pialert to netalterx  incl all devices, network monitor etc
installed via docker

**Comment by jokob-sk** [comment]
Hi @antonov124 ,

Yes, it is. The upgrade should happen almost automatically: https://github.com/jokob-sk/NetAlertX/blob/main/docs/MIGRATION.md

You can also read the upcoming release announcement: https://github.com/jokob-sk/NetAlertX/issues/138

I probably release a new version in a day or two.
Hope this helps,
j

**Reply by antonov124** [reply]
Thx for this quick answer

**Comment by PrzemekSkw** [comment]
Hello, I upgrade and it don't work for me. Then I remove pi.alert and install new container NetAlertX in new location /appdata/netalertx/. What files I can copy from /appdata/pialert to /appdata/netalertx to see my devices?

Regards.

**Reply by jokob-sk** [reply]
Hi there,

Can you have a look at this thread?

https://github.com/jokob-sk/NetAlertX/issues/633
